tensorboard/webapp/plugins/npmi/npmi_plugin.py

```diff
@@ -42,7 +43,7 @@ def _missing_run_error_response(request):
     return _error_response(request, "run parameter is not provided")
 
 
-class NPMIPlugin(base_plugin.TBPlugin):
+class NpmiPlugin(base_plugin.TBPlugin):
     """nPMI Plugin for Tensorboard."""
 
     plugin_name = "npmi"
```

Comment:
This can be a reference to `metadata.PLUGIN_NAME` now which is a bit better to avoid duplication

Reply:
Done in [6f7f82733](https://github.com/tensorflow/tensorboard/pull/3799/commits/6f7f82733fd06403d2f95dd7eb5b0367bc272cb5)